apps/prairielearn/src/models/questions.sql

```diff
@@ -0,0 +1,44 @@
+-- BLOCK questions
```

Comment:
Conventionally we label queries with a verb. In this case, I'd name this `select_questions_for_course`.

Reply:
sure, done.